js/client/modules/@arangodb/testutils/process-utils.js

```diff
@@ -446,6 +448,42 @@ function readImportantLogLines (logPath) {
   return importantLines;
 }
 
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief scans the log files for assert lines
+// //////////////////////////////////////////////////////////////////////////////
+
+function readAssertLogLines (logPath) {
+  const list = fs.list(logPath);
+
+  for (let i = 0; i < list.length; i++) {
+    let fnLines = [];
+
+    if (list[i].slice(0, 3) === 'log') {
+      const buf = fs.readBuffer(fs.join(logPath, list[i]));
+      let lineStart = 0;
+      let maxBuffer = buf.length;
+
+      for (let j = 0; j < maxBuffer; j++) {
+        if (buf[j] === 10) { // \n
+          const line = buf.asciiSlice(lineStart, j);
+          lineStart = j + 1;
+
+          // scan for asesrts from the crash dumper
+          if ((line.search('a7902') !== -1) ||
+              (line.search('c962b') !== -1) ||
+              (line.search('308c3') !== -1)) {
```

Comment:
Not really a fan of this. We could simplify this by just searching for `crash`, I guess.